cmake/config/config.h.in

```diff
@@ -226,6 +226,8 @@
 #cmakedefine INTEL_FPGA_API_VERSION "@INTEL_FPGA_API_VERSION@"
 #cmakedefine INTEL_FPGA_API_HASH    "@INTEL_FPGA_API_HASH@"
 
+#define OPAE_ASE_CFG_SRC_PATH "@CMAKE_CURRENT_SOURCE_DIR@/libopae/plugins/ase/opae_ase.cfg"
```

Comment:
I think what you wanted here was the equivalent of @ASE_INST_SHARE_DIR@, not the source. (See the Python example.) What you have is the source tree. What you want, I hope, is the installation tree.

Reply:
@ASE_INST_SHARE_DIR@ equals to "/share/opae/ase". In Python, you found the current script directory which is concatenated with @ASE_INST_SHARE_DIR@. In FPGA sample execution, we cannot assume the executable's location since it could be anywhere. So it is not easy to get the installation location from executable's location.

Reply:
No. We know exactly where ASE is installed. afu_sim_setup depends on it. I don't understand why this special ASE library can't also make the same inference. It's the only place where we can be confident there is a shared copy opae_ase.cfg that hasn't been installed as a special step by an admin.

Note also that this all has to work on a network of machines with a shared installation. Local disks, like /etc, don't cut it. Home directories require a change by the user.

Reply:
@michael-adler

Reply:
Added "@CMAKE_INSTALL_PREFIX@/share/opae/ase/opae_ase.cfg" to opae_ase.cfg search path.